frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -56,6 +92,59 @@ const AdminDashboard = (): React.ReactElement => {
     setIsDeleteModalOpen(false);
   };
 
+  const previewButtonHandler = (index: number) => {
+    const row: ReviewResponse = data[index]; // The full Review object
+
+    setSelectedReviewTitle(row.books[0].title);
+    setSelectedReviewSubtitle(row.books[0].series?.name ?? "");
+
+    // Authors for the first book concatenated by commas
+    let authors = "";
+    const authorArray = row.books[0].authors.map((author) => author.fullName);
+    const lastAuthorIndex = authorArray.length - 1;
+    authorArray.forEach((authorName, authorIndex) => {
```

Comment:
(minor) We can probably simplify this by using `authorArray.reduce` instead and just splicing to get rid of the extra comma at the end